### ISSUE TITLE
scales

### ISSUE THREAD

**Issue by cringegaming64** [stevenwaterman/musetree#75]
a suggestion, maybe have a little box you can check off if you want the music generated to be in the minor scale or the major scale and maybe the names of the scales too

**Comment by stevenwaterman** [comment]
Sadly since the music generation is handled entirely by MuseNet, and they don't offer that option, It's not possible to add this. As a potential workaround - If you import a MIDI that is in the desired key, MuseNet should try and stick to that.

I'll close this issue as a `wontfix`, but feel free to comment here if you have any other questions, and thanks for the suggestion!

**Comment by farfromsubtle** [comment]
Yeah the model is very easily put into the desired key by providing a short chord progression or melody.
